doc/nrf/links.txt

```diff
@@ -155,6 +155,8 @@
 .. _`known issues for nRF Connect SDK v1.4.0`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/known_issues.html?v=v1-4-0
 
 .. _`connecting to nRF Cloud`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/nrf_cloud.html#connecting
+.. _`updating nRF9160 DK modem firmware`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf9160_gs.html#updating-the-modem-firmware
+.. _`updating Thingy:91 modem firmware`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91_gsg.html#updating-firmware
```

Comment:
```suggestion
```
You don't need to add these links here, as both sections are tagged in the target docs. If you want to link to them, just use the tags (for example, :ref:`updating nRF9160 DK modem firmware <nrf9160_gs_updating_fw_modem>` and :ref:`updating Thingy:91 modem firmware <programming_thingy>`), as suggested above.